src/expression.d

```diff
@@ -4829,12 +4829,12 @@ public:
      * Convert string contents to a 0 terminated string,
      * allocated by mem.xmalloc().
      */
-    final const(char)* toStringz() const
+    extern (D) final const(char)[] toStringz() const
```

Comment:
In this case, what's expected ? Should we just ignore the C++ side, or remove it (or something else) ?